source/fundamentals/poco.txt

```diff
@@ -0,0 +1,249 @@
+{+language+}, is serialized as a BSON ``Int32`` type:
+
+.. code-block:: csharp
+   :copyable: true
+
+   public class House
+   {
+       public ObjectId Id { get; set; }
+
+       [BsonRepresentation(BsonType.Int32)]
+       public char YearBuilt { get; set; }
+   }
+
+Set Field Order
+~~~~~~~~~~~~~~~
+
+The driver serializes properties to BSON fields in the order they
+are specified in the POCO. To store properties in a custom order, you
+can specify the ``Order`` named parameter in the ``BsonElement``
+attribute. The following code shows how you can direct the driver to
+store the ``YearBuilt`` property after the ``Style`` property:
+
+.. code-block:: csharp
+   :copyable: true
+
+   public class House
+   {
+       public ObjectId Id { get; set; }
+
+       [BsonElement(Order = 2)]
+       public int YearBuilt { get; set; }
+
+       [BsonElement(Order = 1)]
+       public string Style { get; set; }
+   }
+
+Any properties that do not have an explicit ``Order`` will occur after
+those that do.
+
+Identify Id Property
+~~~~~~~~~~~~~~~~~~~~
+
+By default, the driver uses any public property named ``Id``, ``id``, or ``_id`` as the
+identifier to map to the BSON ``_id`` field. You can explicitly select the
+identifier using the ``BsonIdAttribute``. The following code shows how
+you can direct the driver to map the ``Identifier`` property to the ``_id`` field:
+
+.. code-block:: csharp
+   :copyable: true
+
+   public class House
+   {
+       [BsonId]
+       public string Identifier { get; set; }
+   }
+
+Omit Empty Fields
+~~~~~~~~~~~~~~~~~
+
+By default, the driver serializes undefined properties to fields with ``null``
+values. You can direct the driver to ignore empty properties using the
+``BsonIgnoreAttribute``. The following code shows how you can prevent
+the driver from serializing the ``YearBuilt`` property if it is empty:
+
+.. code-block:: csharp
+   :copyable: true
+
+   public class House
+   {
+       public ObjectId Id { get; set; }
+
+       [BsonIgnore]
+       public int YearBuilt { get; set; }
+       public string Style { get; set; }
+   }
+
+Example
+-------
+
+The following example shows how the {+driver-short+} inserts a document
+instantiated as a POCO with the following custom field mapping specifications:
+
+- Sets a custom field name ``in_stock`` for the ``instock`` property
+- Serializes the ``price`` field as a BSON ``Double`` type
+
+Define the ``Clothing`` class with the relevant serialization
+attributes:
+
+.. literalinclude:: ../includes/fundamentals/code-examples/poco.cs
+   :start-after: start-model
+   :end-before: end-model
+   :language:  csharp
+   :copyable:
+   :dedent:
+
+Instantiate a ``Clothing`` object and insert the document into a collection:
+
+.. literalinclude:: ../includes/fundamentals/code-examples/poco.cs
+   :start-after: start-insert
+   :end-before: end-insert
+   :language:  csharp
+   :copyable:
+   :dedent:
+
+The BSON representation of the inserted document should resemble the
```

Comment:
Issue: Per the style guide, we try to avoid "should", as it suggests we're doubtful about what our code does :)

```suggestion
The BSON representation of the inserted document looks like this:
```